### PR TITLE
Adding Snippets-Stats scripts

### PR DESCRIPTION
**Pull request by cvalaas**
This adds the scripts (or skeletons of scripts) needed for snippets-stats processing.

---
This change is [Reviewable](https://reviewable.io/reviews/gozer/vertical-etl/16)

### CODE REVIEW

File: nubis/files/confd/templates/snippets-stats.tmpl [hunk 1]
```diff
@@ -0,0 +1,7 @@
+config = {
+  'proxies'            : {"https" : "http://proxy.dmz.scl3.mozilla.com:3128"}
```
Comment (gozer) [inline nubis/files/confd/templates/snippets-stats.tmpl]: In AWS land, the proxies need to be _http://proxy.service.consul:3128_

File: nubis/puppet/snippets_stats.pp [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  require => [
+    File[$virtualenv_path],
+  ],
+}
+
+# Install Mozilla's snippets-stats
+python::pip { 'snippets-stats':
+  ensure     => 'present',
+  virtualenv => "${virtualenv_path}/snippets-stats",
+  url        => 'git+https://github.com/mozilla-it/snippets-stats@78c299062701afcb9df60feb5b91bb5faba74cb8',
```
Comment (gozer) [inline nubis/puppet/snippets_stats.pp]: This repo isn't pip installable, it's missing a setup.py at a minimum to drive installation.

```
$> pip install --user git+https://github.com/mozilla-it/snippets-stats@78c299062701afcb9df60feb5b91bb5faba74cb8
Collecting git+https://github.com/mozilla-it/snippets-stats@78c299062701afcb9df60feb5b91bb5faba74cb8
  Cloning https://github.com/mozilla-it/snippets-stats (to 78c299062701afcb9df60feb5b91bb5faba74cb8) to /tmp/pip-OopbEX-build
  Could not find a tag or branch '78c299062701afcb9df60feb5b91bb5faba74cb8', assuming commit.
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
    IOError: [Errno 2] No such file or directory: '/tmp/pip-OopbEX-build/setup.py'

File: nubis/puppet/salesforce_sfmc.pp [hunk 3]
```diff
@@ -15,7 +15,7 @@
 python::pip { 'sfmc-fetcher':
   ensure     => 'present',
   virtualenv => "${virtualenv_path}/data-integrations",
-  #url        => 'git+https://github.com/mozilla-it/data-integrations@2126a2098c28e3fe447305c2438bd18f4fefa7a1',
+  url        => 'git+https://github.com/mozilla-it/data-integrations@2126a2098c28e3fe447305c2438bd18f4fefa7a1',
```
Comment (gozer) [inline nubis/puppet/salesforce_sfmc.pp]: This repo isn't pip installable, it's missing a setup.py at a minimum to drive installation.
```
$> pip install --user git+https://github.com/mozilla-it/data-integrations@2126a2098c28e3fe447305c2438bd18f4fefa7a1
Collecting git+https://github.com/mozilla-it/data-integrations@2126a2098c28e3fe447305c2438bd18f4fefa7a1
  Cloning https://github.com/mozilla-it/data-integrations (to 2126a2098c28e3fe447305c2438bd18f4fefa7a1) to /tmp/pip-4Bg4wP-build
  Could not find a tag or branch '2126a2098c28e3fe447305c2438bd18f4fefa7a1', assuming commit.
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
    IOError: [Errno 2] No such file or directory: '/tmp/pip-4Bg4wP-build/setup.py'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-4Bg4wP-build/
```

**Comment by gozer** [comment]
This is now conflicting, could you rebase/merge ?

**Comment by cvalaas** [comment]
merged; conflicts resolved

**Pull request merged**